plugins/PaintTool/paint.shader

```diff
@@ -55,10 +54,10 @@ fragment =
         color_index = (color_index << (32 - 1 - u_bitsRangesEnd)) >> 32 - 1 - (u_bitsRangesEnd - u_bitsRangesStart);
 
         vec4 diffuse_color = vec4(u_renderColors[color_index] / 255.0, 1.0);
-        highp float n_dot_l = clamp(dot(normal, light_dir), 0.0, 1.0);
+        highp float n_dot_l = mix(0.3, 0.7, dot(normal, light_dir));
```

Comment:
I've noticed mirrored objects become quite dark -- it's not that important right now but I'll probably start looking here.